ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py

```diff
@@ -781,7 +788,21 @@ def array_indices_axis(
     indices = indices[0]
     if disable_random_axis:
         return [x_dtype, indices_dtype], x, indices
-    return [x_dtype, indices_dtype], x, indices, axis, batch_dims
+
```

Comment:
Please, revert the changes to the strategy, we should ideally create a custom one instead of modifying the array strategy to support an edge case.

Reply:
Okay @CatB1t , I've reverted the changes and created a custom one please check it! Thanks